docs/install/offline.md

```diff
@@ -144,19 +146,18 @@ filesystem mirror without re-building the image.
 
 First, create an empty plugins directory:
 
-```console
+```shell
 mkdir $HOME/plugins
 ```
 
 Next, add a volume mount to docker-compose.yaml:
```

Comment:
```suggestion
Next, add a volume mount to compose.yaml:
```

I didn't know it's `compose.yaml` now - but it looks like [their docs use that too](https://docs.docker.com/compose/intro/compose-application-model/)

should we update our [docker docs](https://coder.com/docs/@matifali-patch-1/install/docker) to match? (I can make that a separate PR)

Reply:
> The default path for a Compose file is compose.yaml (preferred) or compose.yml that is placed in the working directory. Compose also supports docker-compose.yaml and docker-compose.yml for backwards compatibility of earlier versions. If both files exist, Compose prefers the canonical compose.yaml.

From: https://docs.docker.com/compose/intro/compose-application-model/#the-compose-file

Reply:
awesome- let's default to holding by their "preferred" (here, and wherever we can)

the commit I added just adjusts the `vim` command to the filename in the step. I'll add another issue or pr to align the rest of our docs